c_cxx/CMakeLists.txt

```diff
@@ -27,16 +23,25 @@ set(CMAKE_CXX_STANDARD 17)
 
 if(NOT ONNXRUNTIME_ROOTDIR)
   if(WIN32)
-    set(ONNXRUNTIME_ROOTDIR "C:/Program Files (x86)/onnxruntime")
-  else()
-    include_directories("/usr/local/include/onnxruntime")
+    set(ONNXRUNTIME_ROOTDIR "C:/Program Files/onnxruntime")
+else()
+    set(ONNXRUNTIME_ROOTDIR "/usr/local")
   endif()
 endif()
 
+# The ORT package has a different include directory structure to a local install via cmake.
```

Comment:
> The ORT package has a different include directory structure to a local install via cmake.

is this discrepancy by design, or something to fix in ORT?

Reply:
I don't think we generally expect users to be building against ORT source directly. I'd lean towards the examples only using released packages for simplicity. 

It's also a side effect of the old way of enabling additional EPs (which also doesn't seem core to introductory examples) requiring a header to define an exported symbol and that header was in the internal EP directory. I think it would be better to convert any EP we want to use in these examples to use the generic registration function in the C API (e.g. XNNPACK, SNPE and a few other EPs are using that now). That way the user only needs things that come from the /include/onnxruntime path.